src/PhpSpec/CodeGenerator/Writer/TokenizedCodeWriter.php

```diff
@@ -83,9 +83,9 @@ public function insertAfterMethod($class, $methodName, $method)
      */
     private function insertStringAfterLine($target, $toInsert, $line, $leadingNewline = true)
     {
-        $lines = explode("\n", $target);
+        $lines = explode(PHP_EOL, $target);
```

Comment:
if you explode on ``PHP_EOL`` it breaks things when the source code using ``\n`` EOL and you run on windows.